testing/matrix-sdk-test/src/test_json/keys_query_sets.rs

```diff
@@ -650,29 +650,74 @@ impl IdentityChangeDataSet {
     }
 }
 
+/// A set of `/keys/query` responses that were initially created to simulate
+/// when a user that was verified reset his keys and became unverified.
+///
+/// The local user (as returned by [`PreviouslyVerifiedTestData::own_id`]) is
+/// `@alice:localhost`. There are 2 other users: `@bob:localhost` (returned by
+/// [`PreviouslyVerifiedTestData::bob_id`]), and `@carol:localhost` (returned by
+/// [`PreviouslyVerifiedTestData::carol_id`]).
+///
+/// We provide two `/keys/query` responses for each of Bob and Carol: one signed
+/// by Alice, and one not signed.
+///
+/// Bob and Carol each have 2 devices, one signed by the owning user, and
+/// another one not cross-signed.
+///
+/// The `/keys/query` responses were generated using a local synapse.
 pub struct PreviouslyVerifiedTestData {}
 
 #[allow(dead_code)]
 impl PreviouslyVerifiedTestData {
+    /// Secret part of Alice's master cross-signing key.
+    ///
+    /// Exported from Element-Web with the following console snippet:
+    ///
+    /// ```javascript
+    /// (await mxMatrixClientPeg.get().getCrypto().olmMachine.exportCrossSigningKeys()).masterKey
+    /// ```
     pub const MASTER_KEY_PRIVATE_EXPORT: &'static str =
```

Comment:
Would it be worth it to prepend `ALICE` to these key constants? Or maybe reuse the same `own` naming convention

Reply:
probably. I was trying to avoid getting into doing much beyond documenting the current behaviour here.